sycl/doc/extensions/ExplicitSIMD/dpcpp-explicit-simd.md

```diff
@@ -0,0 +1,731 @@
+# Explicit SIMD Programming Extension for DPC++
+
+## Introduction
+
+The main motivation for introducing the Explicit SIMD Programming (ESP) DPC++
+extension is enabling low-level efficient programming for Intel Gen
+architectures. More specifically, Explicit SIMD provides the following
+additional features:
+- Manual vectorization of device code using the `simd` class mapped to Gen's
+  general register file. This allows to write efficient code not relying on
+  further widening by the compiler, as with traditional SIMT programming.
+- Low-level APIs efficiently mapped to Gen architecture, such as block reads.
+
+One of the future directions is enabling this extension for other architectures,
+such as x86, with extracting and clearly marking generic and target-dependent
+API portions.
+
+## Explicit SIMD execution model
+
+Explicit SIMD execution model is a basically an equivalent of the base SYCL
+execution model with subgroup size restricted to 1. Which means in ESP
+it is guaranteed that there is a single work item in any workgroup, and it maps
```

Comment:
You are right, this is just a typo, obviously.

Reply:
Ok, thanks for the clarification.  I genuinely wasn't sure what the restrictions of the initial implementation were!

It might be useful to add a simple example showing the mapping between an ND-range and the number of thread-groups and EU threads, and showing the usage of explicit SIMD together with work-group barriers and SLM.  That would help to highlight what's possible with this extension.

Reply:
ok, will try. Will add a TODO for now.